tests/ui_tools/test_messages.py

```diff
@@ -1593,6 +1593,96 @@ def test_transform_content(self, mocker, raw_html, expected_content):
         rendered_text = Text(content)
         assert rendered_text.text == expected_content
 
+    @pytest.mark.parametrize(
+        "raw_html, expected_message_links",
+        [
+            (
+                """
+                <p><a href="https://github.com/zulip/zulip-terminal/pull/1">https://github.com/zulip/zulip-terminal/pull/1</a></p>
```

Comment:
Should we have some link where the text is not the same as the link?
One of the first two?

Reply:
Done :)